doc/source/installation.rst

```diff
@@ -10,17 +10,40 @@ Environment
 Installation
 ============
 
+Since there is no stable release yet, the only option is to install the project from the source.
 
-1. Install http://github.com/tendrl/bridge_common
-2. Install http://github.com/tendrl/gluster_bridge
+Development version from the source
+-----------------------------------
 
-3. At the command line::
+1. Install http://github.com/tendrl/bridge_common from the source code::
+
+    $ git clone https://github.com/Tendrl/bridge_common.git
+    $ cd bridge_common
+    $ mkvirtualenv ceph_bridge
```

Comment:
Oh yeah :) Copy-Paste issue. Will correct it as below

mkvirtualenv bridge_common